src/ape_plugins/_cli.py

```diff
@@ -116,26 +117,49 @@ def add(cli_ctx, plugin, version, skip_confirmation):
         subprocess.call([sys.executable, "-m", "pip", "install", "--quiet", plugin])
 
 
+def _get_config_error() -> ConfigError:
+    expected_format = "plugins:\n  - name: <plugin-name>\n    version: <plugin-version>  # optional"
+    return ConfigError(f"Config item mis-configured. Expected format:\n\n{expected_format}\n")
+
+
+def _get_plugin_names(item: Dict) -> Tuple[str, str]:
+    """
+    Extracts the module name and package name from the configured
+    plugin. The package name includes `==<version>` if the version is
+    specified in the config.
+    """
+    try:
+        name = item["name"]
+        module_name = f"ape_{name.replace('-', '_')}"
+        package_name = f"ape-{name}"
+        version = item.get("version")
+
+        if version:
+            package_name = f"{package_name}=={version}"
```

Comment:
`package_name` doesn't quite match what the purpose is, maybe something more like `package_install_name`